packages/core/telemetry/server/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Same comment we should have only one package.json per package (@strapi/telemetry) is one single package